adiv5-swd.rb

```diff
@@ -54,7 +54,7 @@ def read(port, addr, opt={})
     if !opt[:count]
       reply = @drv.transfer(req)
     else
-      req[:count] = readcount
```

Comment:
Probably we should change the bitbang driver to accept `:count` instead of `:val`